Makefile

```diff
@@ -746,7 +746,10 @@ endif
 roxvet: $(ROXVET_BIN)
 	@echo "+ $@"
 	@# TODO(ROX-7574): Add options to ignore specific files or paths in roxvet
-	$(SILENT)go vet -vettool "$(ROXVET_BIN)" $(shell go list -e ./... | grep -v 'operator/pkg/clientset')
+	$(SILENT)go vet -vettool "$(ROXVET_BIN)" $(shell go list -e ./central/...)
+	$(SILENT)go vet -vettool "$(ROXVET_BIN)" $(shell go list -e ./migrator/...)
+	$(SILENT)go vet -vettool "$(ROXVET_BIN)" $(shell go list -e ./pkg/...)
+	$(SILENT)go vet -vettool "$(ROXVET_BIN)" $(shell go list -e ./... | grep -v -e '^github.com/stackrox/rox/central' -e '^github.com/stackrox/rox/migrator' -e '^github.com/stackrox/rox/pkg' -e '^github.com/stackrox/rox/operator/pkg/clientset')
```

Comment:
Would that work?
```Makefile
$(SILENT)go list -e ./... | grep -v 'operator/pkg/clientset' | xargs -n 100 go vet -vettool "$(ROXVET_BIN)"
```

Reply:
That was my previous idea (limiting to a thousand files rather than a hundred).
It raised the concern that some files could escape the vetter.

I did some tests with `xargs -n 100` and it seems to iterate up to exhaustion of the input list. I'll probably revert to the xargs variant (more future-proof I think).